### PR TITLE
Tests: Add static indexes to fill out 0.20.x coverage.

### PR DESCRIPTION
**Pull request by rjernst**
NOTE: most of the the create-bwc-index.py changes are already on master, this just tweaks them for 1.x to skip the completion field in older 0.20.x indexes.

### CODE REVIEW

File: dev-tools/create-bwc-index.py [hunk 1]
```diff
@@ -136,11 +136,39 @@ def generate_index(client):
...
+    mappings['analyzer_type1'] = {
+      'analyzer': 'standard',
+      'properties': {
+        'string_with_index_analyzer': {
+          'type': 'string',
+          'index_analyzer': 'standard'
+        },
+      }
+    }
+    if not version.startswith('0.20') or version == '0.20.6':
```
Comment (rmuir) [inline dev-tools/create-bwc-index.py]: as someone who doesnt know python, i find this hard to grok. can we simplify this condition or add comments?

Comment (rjernst) [inline dev-tools/create-bwc-index.py]: This was a goof.  I've fixed it to just be the first part.  Anything on 0.20 didn't have the completion suggester.

**Comment by rjernst** [comment]
@rmuir I pushed an update, if you can take another look.

**Comment by rmuir** [comment]
+1

**Pull request merged**